OpenRA.Utility/App.config

```diff
@@ -1,11 +1,11 @@
-<?xml version="1.0" encoding="utf-8" ?>
-<configuration>
-  <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1" appliesTo="v1.0.3705">
-      <dependentAssembly>
-        <assemblyIdentity name="Microsoft.Build.Framework" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-100.0.0.0" newVersion="12.0.0.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <runtime>
```

Comment:
Are the differences between this `.config` and the OpenRA.Game one important?
If not, can we please make them consistent?